lib/buildkite.go

```diff
@@ -0,0 +1,16 @@
+package lib
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// BuildkiteAgentHook returns the path to the given hook on disk
+func BuildkiteAgentHook(hookName string) string {
+	return filepath.Join(BuildkiteAgentHooksDir, hookName)
+}
+
+// CiStudioCommonAgentHook returns the path to the given hook on disk
+func CiStudioCommonAgentHook(hookName string) string {
```

Comment:
[nit] conventional here would be `CIStudioCommonAgentHook`, but 🙈 (see https://github.com/golang/go/wiki/CodeReviewComments#initialisms)